### ISSUE TITLE
About the stage of pre-training

### ISSUE THREAD

**Issue by WADE1121** [antoyang/FrozenBiLM#12]
Hi, I noticed that you only trained on webvid10m for two epochs, is it converging already?

**Comment by antoyang** [comment]
Hi, if I remember correctly, the loss was plateauing after the second epoch.
